lib/widgets/product_listing.dart

```diff
@@ -53,18 +53,27 @@ class _ProductListingState extends State<ProductListing> {
   ) {
     List<Widget> widgets = List<Widget>();
 
-    widgets.add(_buildCategoryFilterMenu());
     snapshots.asMap().forEach((index, data) {
       _addProductToWidgetListInDisplay(data, widgets, context, index);
     });
 
-    return ListView(
+    return Column(
       key: Key(makeTestKeyString(
         TKUsers.admin,
         TKScreens.productListing,
         "list",
       )),
-      children: widgets,
+      children: [
+        _buildCategoryFilterMenu(),
+        Expanded(
+          child: ListView.builder(
```

Comment:
Since the widget list is already being built dynamically by the `_addProductToWidgetListInDisplay` method, I think the final list creation can be simplified to 
```
Expanded(
  child: ListView(
    children: widgets,
  ),
)
```
Please let me know your thoughts. @DavidCuentasMar

Reply:
Nice, I didn't know this was possible. Thanks

Reply:
Nice, I didn't know that was possible. Thanks.